pkgs/top-level/python-packages.nix

```diff
@@ -11650,6 +11650,10 @@ in {
     };
   };
 
+  meliae = if ! isPy3k then
```

Comment:
use `disabled = isPy3k;` in the expression instead

Reply:
Fixed, thanks. I've squashed both changes and --force pushed.